site/content/en/guides/consumer/function/export/github-actions/_index.md

```diff
@@ -82,11 +92,22 @@ jobs:
           - name: Run all kpt functions
             uses: docker://gcr.io/kpt-dev/kpt:latest
             with:
-                args: fn run exmaple-package
+                args: fn run example-package
+          
+          - name: Show diff
```

Comment:
Is the diff step optional? In other words, does `export` command generate it?

Reply:
Technically speaking, a `diff` step is not the duty of `kpt fn export`. It is the user's call whether to add the step **manually** or not.